src/main/resources/application.properties

```diff
@@ -0,0 +1,16 @@
+spring.application.name=backend-spring-boot-tech-it-easy-controller
+
+spring.sql.init.platform=postgres
+spring.datasource.url=jdbc:postgresql://localhost:5432/database_springboot
+spring.datasource.username=postgres
+spring.datasource.password=springboot
```

Comment:
Later zul je even een manier moeten vinden om dit te houden. Je kan een
.env file gebruiken. Kijk ook even op mijn repo of bericht me een keer als je wilt weten hoe dat kan.

Reply:
Dit is een goede toevoeging van Maarten, maar voor je eindopdracht niet nodig